br/pitr-known-issues.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Known Issues in Log Backup
+summary: Learn known issues in log backup.
+---
+
+# Known Issues in Log Backup
+
+This document lists the known issues and corresponding workarounds when you use the log backup feature.
+
+## BR encounters the OOM problem when you execute PITR recovery or the `br log truncate` command
+
+Issue: [#36648](https://github.com/pingcap/tidb/issues/36648)
+
+Consider the following possible causes.
+
+- PITR recovery experiences OOM because the log interval to be recovered is too large.
+    It is recommended that the recovery log interval range from two days to one week. That is, perform a full backup operation at least once in two days or once in up to one week during the backup process.
+- OOM occurs when you delete a log because the interval of the deleted log is too large.
+    To resolve this issue, you can reduce the interval for the log to be deleted. You can delete small interval logs several times instead of deleting the large interval logs directly.
+- The memory allocation of the node where the BR process is located is too low.
+    It is recommended to scale up the node memory configuration to at least 16 GB to ensure that PITR has sufficient memory resources for recovery.
+
+## The upstream database uses TiDB Lightning Physical Mode to import data, which makes it impossible to use the log backup feature
+
+Currently the log backup feature is not fully adapted to TiDB Lightning, so the data imported by TiDB Lightning Physical Mode cannot be backed up to the log.
+
+In upstream clusters where you create log backup tasks, avoid using the TiDB Lightning Physical Mode to import data. Instead, you can use TiDB Lightning Logical Mode. If you do need to use the Physical Mode, you can do a full backup operation after the import is complete, so that PITR can be restored to the time point after the full backup.
```

Comment:
```suggestion
In upstream clusters where you create log backup tasks, avoid using the TiDB Lightning physical mode to import data. Instead, you can use TiDB Lightning logical mode. If you do need to use the physical mode, perform a full backup after the import is complete, so that PITR can be restored to the time point after the full backup.
```